### PR TITLE
Use PR number, job name and job id as packet server tags 

### PR DESCRIPTION
**Pull request by akiselev1**

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh [hunk 1]
```diff
@@ -40,6 +42,7 @@ cat > packet-setup.yaml <<-EOF
         facility: any
         wait_for_public_IPv: 4
         state: active
+        tags: "{{ lookup('env', 'PROW_JOB_ID') }}"
```
Comment (andfasano) [inline ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh:45]: On a second review, probably it's better to add the following env vars:
- `PULL_NUMBER`
- `JOB_NAME`
With `BUILD_ID` already present in the hostname, a person will have enough (required) information to rebuild the original job link

Comment (akiselev1) [reply ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh]: Done

**Comment by akiselev1** [comment]
https://github.com/openshift/release/pull/10913 - done, waiting for approval.

**Pull request merged**